Minor updates to Remote Access (#3622)
raspberrypi/documentation: documentation/asciidoc/computers/remote-access/ssh.adoc

. Select *Enabled* next to *SSH*.
. Click *OK*.

==== While flashing a fresh OS image

To configure SSH on a completely new installation of Raspberry Pi OS:

> . Follow the instructions in thexref:computers/getting-started.adoc#install-using-imager[Install with Imager] guide.

Author: ```suggestion
. Follow the instructions in the xref:computers/getting-started.adoc#install-using-imager[Install with Imager] guide.
```

> . Follow the instructions in the xref:computers/getting-started.adoc#install-using-imager[Install with Imager] guide.

Author: ```suggestion
. Follow the instructions in the xref:../computers/getting-started.adoc#install-using-imager[Install with Imager] guide.
```
D'oh

> . Follow the instructions in the xref:../computers/getting-started.adoc#install-using-imager[Install with Imager] guide.

Author: ```suggestion
. Follow the instructions in the xref:../computers/getting-started.adoc#raspberry-pi-imager[Install with Imager] guide.
```
